robottelo/vm.py

```diff
@@ -269,10 +268,9 @@ def install_katello_agent(self):
         result = self.run('rpm -q katello-agent')
         if result.return_code != 0:
             raise VirtualMachineError('Failed to install katello-agent')
-        if bz_bug_is_open('1431747'):
-            gofer_start = self.run('service goferd start')
-            if gofer_start.return_code != 0:
-                raise VirtualMachineError('Failed to start katello-agent')
+        gofer_start = self.run('service goferd start')
```

Comment:
Not sure that after I read both bugs description I see confirmation that goferd should not start right away after installation is finished
Yes, there is a chance that you should have some delay, but it is wrong approach to start service manually
Correct me if I am wrong, till that moment NACK on that fix